src/magnify/apps/core/views/user.py

```diff
@@ -0,0 +1,87 @@
+"""View to manage users"""
+
+from django.contrib.auth.password_validation import validate_password
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.utils.translation import gettext_lazy as _
+
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.views import APIView, Response
+
+from magnify.apps.core.models import User
+from magnify.apps.core.serializers import (
+    UserCreateResponseErrorSerializer,
+    UserCreateResponseSuccessSerializer,
+    UserCreateSerializer,
+)
+
+
+class UserView(APIView):
```

Comment:
Consider using a ModelViewset to not implement everything https://www.django-rest-framework.org/api-guide/viewsets/#modelviewset